src/cli/common.js

```diff
@@ -197,3 +202,22 @@ function loadPluginBudget(baseDir: string): Promise<Budget | null> {
   const budgetPath = pathJoin(baseDir, "config", "pluginBudgets.json");
   return loadJsonWithDefault(budgetPath, pluginBudgetParser, () => null);
 }
+
+/**
+ * Load the currency details from config, falling back on defaults
+ * if need be.
+ */
+export async function loadCurrencyDetails(
+  currencyDetailsPath: string
+): Promise<CurrencyDetails> {
+  const {name, suffix} = await loadJsonWithDefault(
+    currencyDetailsPath,
+    currencyConfigParser,
+    () => ({
+      name: DEFAULT_NAME,
+      suffix: DEFAULT_SUFFIX,
+    })
```

Comment:
Please export a  `defaultCurrencyConfig()` function from `currencyConfig.js`. That way it's possible to add a new field and update the default all in one place.

Reply:
Done, thanks for suggestion, much cleaner.